src/abstractdataframe/io.jl

```diff
@@ -178,8 +177,14 @@ end
         write(io, "<th>$column_name</th>")
     end
     write(io, "</tr>")
-    tty_rows, tty_cols = _displaysize(io)
-    mxrow = min(n,tty_rows)
+    haslimit = get(io, :limit, true)
+    n = size(df, 1)
+    if haslimit
+        tty_rows, tty_cols = displaysize(io)
```

Comment:
Yeah, but adding an underscore doesn't hurt, and we could imagine backporting things. If we remove 0.4 support, we should remove `_displaysize` too, not just stop using it.